### PR TITLE
Oracle on introduction page in v6

### PR DESCRIPTION
**Pull request by hjamil-24**

### CODE REVIEW

File: versioned_docs/version-6.x.x/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ sidebar_label: Introduction
...
+Sequelize is a promise-based [Node.js](https://nodejs.org/en/about/) [ORM tool](https://en.wikipedia.org/wiki/Object-relational_mapping) for [Oracle](https://en.wikipedia.org/wiki/Oracle_Database),[Postgres](https://en.wikipedia.org/wiki/PostgreSQL), [MySQL](https://en.wikipedia.org/wiki/MySQL), [MariaDB](https://en.wikipedia.org/wiki/MariaDB), [SQLite](https://en.wikipedia.org/wiki/SQLite), [Microsoft SQL Server](https://en.wikipedia.org/wiki/Microsoft_SQL_Server), [Amazon Redshift](https://docs.aws.amazon.com/redshift/index.html) and [Snowflake’s Data Cloud](https://docs.snowflake.com/en/user-guide/intro-key-concepts.html). It features solid transaction support, relations, eager and lazy loading, read replication and more.
```
Comment (WikiRik) [inline versioned_docs/version-6.x.x/index.md:19]:
```suggestion
Sequelize is a promise-based [Node.js](https://nodejs.org/en/about/) [ORM tool](https://en.wikipedia.org/wiki/Object-relational_mapping) for [Postgres](https://en.wikipedia.org/wiki/PostgreSQL), [MySQL](https://en.wikipedia.org/wiki/MySQL), [MariaDB](https://en.wikipedia.org/wiki/MariaDB), [SQLite](https://en.wikipedia.org/wiki/SQLite), [Microsoft SQL Server](https://en.wikipedia.org/wiki/Microsoft_SQL_Server), [Oracle](https://en.wikipedia.org/wiki/Oracle_Database), [Amazon Redshift](https://docs.aws.amazon.com/redshift/index.html) and [Snowflake’s Data Cloud](https://docs.snowflake.com/en/user-guide/intro-key-concepts.html). It features solid transaction support, relations, eager and lazy loading, read replication and more.
```

File: versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md [hunk 2]
```diff
@@ -224,6 +224,15 @@ Sequelize also lets you pass credentials in URL format:
 const sequelize = new Sequelize('oracle://user:pass@hostname:port/servicename');
 ```
 
+You can pass an Easy Connect String, a Net Service Name, or a Connect Descriptor to the Sequelize constructor using `dialectOptions.connectString`:
+
+```js
+const sequelize = new Sequelize({dialect: 'oracle', username: 'user', password: 'password', dialectOptions: {connectString: 'inst1'}});
```
Comment (WikiRik) [inline versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md:230]:
```suggestion
const sequelize = new Sequelize({
    dialect: 'oracle',
    username: 'user',
    password: 'password',
    dialectOptions: {
        connectString: 'inst1'
     }
 });
```
Just for readability

**Review by WikiRik — COMMENTED** [review]
Two suggestions, but apart from that it looks good. Thanks!

File: versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md (hunk 2 above)
Comment (ephys) [reply versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md:230]: There is an identation issue in your suggestion (penultimate bracket), we also use 2 spaces in the rest of our codebase :)

Comment (WikiRik) [reply versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md:230]: Good catch, fixed

File: versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md [hunk 3]
```diff
@@ -224,6 +224,15 @@ Sequelize also lets you pass credentials in URL format:
...
+```
+Note that the `database`, `host` and `port` will be overriden and the values in connectString will be used for authentication.
+
+Please refer to [Connect String](https://oracle.github.io/node-oracledb/doc/api.html#connectionstrings) for more about connect strings.
```
Comment (sharadraju) [inline versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md:234]: Please change the link to [https://node-oracledb.readthedocs.io/en/latest/user_guide/connection_handling.html#connectionstrings](https://node-oracledb.readthedocs.io/en/latest/user_guide/connection_handling.html#connectionstrings)

File: versioned_docs/version-6.x.x/index.md (hunk 1 above)
Comment (sharadraju) [inline versioned_docs/version-6.x.x/index.md:19]: Please change `Oracle` to `Oracle Database`

File: versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md [hunk 4]
```diff
@@ -227,11 +227,18 @@ const sequelize = new Sequelize('oracle://user:pass@hostname:port/servicename');
...
 
 ```js
-const sequelize = new Sequelize({dialect: 'oracle', username: 'user', password: 'password', dialectOptions: {connectString: 'inst1'}});
+const sequelize = new Sequelize({
+    dialect: 'oracle',
+    username: 'user',
+    password: 'password',
+    dialectOptions: {
+      connectString: 'inst1'
+    }
```
Comment (WikiRik) [inline versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md:236]:
```suggestion
  dialect: 'oracle',
  username: 'user',
  password: 'password',
  dialectOptions: {
    connectString: 'inst1'
  }
```

Comment (hjamil-24) [reply versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md]: Done.

**Review by WikiRik — APPROVED** [review]

**Pull request merged**